pkg/config/controller_config.go

```diff
@@ -88,3 +99,38 @@ func configInit(metadata EC2Metadata) error {
 
 	return nil
 }
+
+// try to find cluster name, search in env then in ec2 instance tags
+// returns empty string if not found
+func getClusterName(sess *session.Session) (string, error) {
+	cn := os.Getenv(CLUSTER_NAME)
+	if cn != "" {
+		return cn, nil
```

Comment:
probably a good idea to add some logging here that name is being fetched from the env variable

Reply:
I dont think logging is important here, as long as variable is resolved and resolved to correct name it's not important where it comes from. If variable is in wrong value, we still log entire config, customer can see what actual value is. We should specify variables resolution order in the documentation.